Tests/SentryTests/Performance/SentryTracerTests.swift

```diff
@@ -242,6 +242,12 @@ class SentryTracerTests: XCTestCase {
         let child2 = sut.startChild(operation: fixture.transactionOperation)
         let child3 = sut.startChild(operation: fixture.transactionOperation)
         child2.finish()
+        
+        //Without this sleep sut.timestamp and child2.timestamp sometimes
+        //are equal. We need to make sure that SentryTracer is not changing
+        //the timestamp value of proper finished spans.
+        Thread.sleep(forTimeInterval: 0.1)
```

Comment:
`m`: Can't we achieve the same by changing the time of the dateprovider instead with something like `fixture.dateProvider.setDate(date: fixture.dateProvider.date().addingTimeInterval(bySeconds))`?

Reply:
Ah now I get it thanks.